packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -459,3 +604,38 @@ export class DatabaseCluster extends DatabaseClusterBase {
 function databaseInstanceType(instanceType: ec2.InstanceType) {
   return 'db.' + instanceType.toString();
 }
+
+/**
+ * Get bucket and object-level ARN's for specified S3 buckets
+ */
+function bucketAndObjectArns(buckets: s3.IBucket[]): string[] {
+  const arns: string[] = [];
+  for (const bucket of buckets) {
+    arns.push(bucket.bucketArn);
+    arns.push(bucket.arnForObjects('*'));
+  }
+  return arns;
+}
+
+/**
+ * Get default parameter group family for given engine and version
+ */
+function getClusterParameterGroupFamily(engine: DatabaseClusterEngine, engineVersion?: string): string {
+  if (engine === DatabaseClusterEngine.AURORA) {
+    return 'aurora5.6';
+  } else if (engine === DatabaseClusterEngine.AURORA_MYSQL) {
+    return 'aurora-mysql5.7';
+  } else if (engine === DatabaseClusterEngine.AURORA_POSTGRESQL) {
+    if (engineVersion) {
+      if (engineVersion.startsWith('9')) {
+        return "aurora-postgresql9.6";
+      } else if (engineVersion.startsWith('10')) {
+        return "aurora-postgresql10";
+      } else if (engineVersion.startsWith('11')) {
+        return "aurora-postgresql11";
+      }
+    }
+    return "aurora-postgresql11";
+  }
+  throw new Error(`Unknown database engine: ${engine.name}`);
+}
```

Comment:
This doesn't feel like the right place for this. It should go into `DatabaseClusterEngine` and `DatabaseInstanceEngine`.